mantis-examples/mantis-examples-sine-function/build.gradle

```diff
@@ -27,6 +27,6 @@ task execute(type:JavaExec) {
     classpath = sourceSets.main.runtimeClasspath
 }
 dependencies {
-    implementation project(':mantis-server:mantis-server-worker')
+    implementation project(':mantis-runtime-executor')
```

Comment:
I think we should remove this completely and have it part of runtime.

So, essentially, we will have a runtime loader, which has the API to load the runtime and then the runtime itself.